.ci/Jenkinsfile

```
@@ -19,14 +21,15 @@ pipeline {
     disableResume()
     durabilityHint('PERFORMANCE_OPTIMIZED')
     quietPeriod(10)
-    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
+    rateLimitBuilds(throttle: [count: 4, durationName: 'hour', userBoost: true])
   }
   triggers {
     issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
+    upstream('Beats/beats/master')
```

Otherwise, it will run on a PR basis, the below approach matches the same one as defined in :
- https://github.com/elastic/beats/pull/19490

```suggestion
    // disable upstream trigger on a PR basis
    upstream("Beats/beats/${ env.JOB_BASE_NAME.startsWith('PR-') ? 'none' : env.JOB_BASE_NAME }")
```